requirements.txt

```diff
@@ -1,3 +1,3 @@
-Markdown
+Markdown==3.4.1
```

Comment:
found this value in your Dockerfile; not sure if this is correct. Maybe you want latest here and in Dockerfile's `MARKDOWN_VER` ARG too?

Reply:
I prefer not to set the Markdown version as the plugin works with many of them.
In the `Dockerfile` and `docker-compose.yml` there is a version but it's just a default, in case it hasn't been set externally with something like `export MARKDOWN_VER=3.5.2`.